v1.0/json-generator/consts.py

```diff
@@ -66,6 +66,17 @@ def __init__(self, genome=GENOME, read_length=READ_LENGTH, with_ercc=False, umis
             self.rsem_dir = "/data/reddylab/Reference_Data/Genomes/mm9/RSEM/RSEM_genome"
             self.genome_fasta_files = ["/data/reddylab/Reference_Data/Genomes/mm9/mm9.fa"]
             self.regions_bed_file = "/data/reddylab/Reference_Data/Genomes/mm9/mm9.chrom.bed"
+        elif _genome == 'mm39' or _genome == 'grcm39':
+            self.bowtie_genome_ref_first_index = "/data/reddylab/Reference_Data/IGVF/GRCm39/bowtie_GRCm39/IGVFFI928QLXO_GRCm39.1.ebwt"
+            self.bowtie2_genome_ref_first_index = "/data/reddylab/Reference_Data/IGVF/GRCm39/bowtie2_GRCm39/IGVFFI928QLXO_GRCm39.1.bt2"
+            self.star_genome_dir = "/data/reddylab/Reference_Data/IGVF/GRCm39/STAR_GRCm39_genome"
+            self.genome_sizes_file = "/data/reddylab/Reference_Data/IGVF/GRCm39/Gencode/mm39.chrom.sizes"
+            self.encode_blacklist_bedfile = "/data/reddylab/Reference_Data/ENCODE/mm10.blacklist.bed" #TODO: Create liftover from mm10 to mm39
```

Comment:
I have added a exclusion list from a recent publication that should be used here:
```
/data/reddylab/Reference_Data/IGVF/GRCm39/mm39.excluderanges.bed
```
Please add that one.